wordssite/words/migrations/0002_choice_word_word_translation_delete_translation_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.0.4 on 2022-05-03 14:49
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('words', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Choice',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('choice_text', models.CharField(max_length=200)),
+                ('votes', models.IntegerField(default=0)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='word',
+            name='word_translation',
+            field=models.CharField(default='DEFAULT VALUE', max_length=200),
+        ),
+        migrations.DeleteModel(
```

Comment:
I actuall thought it is a good idea to have a Translation model. Then the word could have multiple translations. And then we would need a Language model, so each Translation could have one language.